.compatibility_tests/compatibility_test_1_0/Cargo.toml

```diff
@@ -19,12 +19,15 @@ publish = false
 [workspace]
 
 [dependencies]
-test_roqoqo_1_0 = {package = "roqoqo", version="=1.0.0"}
-test_roqoqo_derive_1_0 = {package = "roqoqo-derive", version="=1.0.0"}
-qoqo_calculator = { version="1.1" }
-roqoqo = {version="1.11.0", path="../../roqoqo", features=["serialize", "overrotate"]}
-bincode = {version="1.3"}
+test_roqoqo_1_0 = { package = "roqoqo", version = "=1.0.0" }
+test_roqoqo_derive_1_0 = { package = "roqoqo-derive", version = "=1.0.0" }
+qoqo_calculator = { version = "1.1" }
```

Comment:
This should now be 1.2, here and everywhere (I will only add the comment here, but please replace this in the whole repo)